src/redux/selectors/data/bookingsSelectors.js

```diff
@@ -76,16 +76,19 @@ export const selectUpComingBookings = createSelector(
   state => state.data.offers,
   state => state.data.stocks,
   (bookings, offers, stocks) => {
-    const sevenDaysFromNow = moment()
-      .clone()
-      .add(SLIDING_DAYS, 'days')
+    const sevenDaysFromNow = moment().clone().add(SLIDING_DAYS, 'days')
 
     const filteredBookings = bookings.filter(booking => {
-      if (booking.isCancelled || booking.isUsed) return false
+      const filteredStock = selectStockById({ data: { stocks } }, booking.stockId)
```

Comment:
```suggestion
      const idoineStock = selectStockById({ data: { stocks } }, booking.stockId)
```
En fait je n'ai pas d'idée de nom, mais je trouve que le nom filteredStock évoque une liste, même sans 'S', ça m'a induit en erreur.
Ou au pire juste `stock`, on voit qu'on le sélectionne via l'ID référencé dans le booking.